authentication-ldap/src/main/java/com/synopsys/integration/alert/authentication/ldap/action/LdapManager.java

```diff
@@ -59,9 +63,8 @@ public Optional<LdapAuthenticationProvider> getAuthenticationProvider() throws A
 
     public Optional<LdapAuthenticationProvider> createAuthProvider(LDAPConfigModel ldapConfigModel) throws AlertConfigurationException {
         try {
-            System.out.println(ldapConfigModel.getEnabled());
-            Boolean ldapEnabled = ldapConfigModel.getEnabled();
-            if (null != ldapEnabled && !ldapEnabled) {
+            Boolean ldapEnabled = ldapConfigModel.getEnabled().orElse(false);
+            if (!ldapEnabled) {
```

Comment:
Are we sure ldapEnabled cant be null (aka required)? NPE will happen here if it is

Reply:
getEnabled() returns an Optional.empty(), and the orElse sets it to false